### ISSUE TITLE
JUnit dependency in runtime classpath

### ISSUE THREAD

**Issue by hodgdon** [touchlab/Kermit#385]
`implementation(kotlin("test-junit"))` should be applied to test configuration rather than main.
https://github.com/touchlab/Kermit/blob/main/kermit/build.gradle.kts#L100C17-L100C53

**Comment by ptitjes** [comment]
Duplicates of #383.

@kpgalligan Is there a reason the main source-set depends on JUnit?
This is blocking us from moving to 2.0 version.

**Comment by Phaestion** [comment]
This is a long running issue, we have an exclude rule in our dependency specification for this, it causes our gradle build and tests to fail.
FYI this is our exlude statement:
```
implementation("co.touchlab:kermit:2.0.2") {
    exclude(group = "org.jetbrains.kotlin", module = "kotlin-test-junit")
}
```

**Comment by findjigar** [comment]
Fixed in version 2.0.3
